.travis.yml

```diff
@@ -0,0 +1,29 @@
+dist: xenial
+sudo: true
+
+language: python
+python:
+  - 3.5
+  - 3.6
+  - 3.7
+
+# Command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
+install: pip install -U tox-travis
```

Comment:
Deployment is done via CircleCI ATM, not sure this travis config is necessary.

If there's a reason to add travis, please split on a different PR.

Reply:
removed